sep-template.md

```diff
@@ -14,6 +14,9 @@ Discussion: <link to where discussion for this SEP is taking place, typically th
 "If you can't explain it simply, you don't understand it well enough." Provide a simplified and
 layman-accessible explanation of the SEP.
 
+## Dependencies
+Should clearly list the dependencies this SEP has on other SEPs or CAPs. May go into further discussion about the circumstances of the dependency, although a list sufficient.
+
```

Comment:
a list "is" sufficient

Reply:
Thanks, fixed in 4f7b7f9f9877d37af7f8e481c903e3fe4d350b7f.